src/rust/engine/fs/src/directory.rs

```diff
@@ -811,6 +929,32 @@ impl From<&DigestTrie> for remexec::Tree {
   }
 }
 
+pub struct DigestTrieDiff {
```

Comment:
```suggestion
#[derive(Default)]
pub struct DigestTrieDiff {
```

If you derive `Default` (it can be derived for any struct made up of fields that implemented `Default`), then you can construct an empty instance of the class (on line 556) using: `let diff = DigestTrieDiff::default()`.